my-code/whats-in/src/pages/Movie/index.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import iconBack from '../../images/icon-arrow-grey.svg'
+import logoImdb from '../../images/logo-imdb.svg'
+import logoRottenTomatoes from '../../images/logo-rotten-tomatoes.svg'
+import iconLike from '../../images/icon-heart-grey.svg'
+import iconLikeWhite from '../../images/icon-heart-white.svg'
+
+export default function Moviepage( {movie, isMovieLiked, handleLike, likedMovies} ) {
+    const navigation = useNavigate();
+
+    const handleNavigation = () => {
+        navigation(-1)
```

Comment:
nothing guarantees that the user came from the homepage. with this route-based approach, it could have opened directly the movie detail.